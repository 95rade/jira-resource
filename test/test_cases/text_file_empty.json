{
  "input":{
    "params":{
      "summary_file":"resources/sample-empty.txt",
      "description_file":"resources/sample-empty.txt",
      "issue_type":"Feature"
    },
    "source":{
      "url":"https://jira.url",
      "username":"some_user",
      "password":"awesome_pass",
      "project":"PROJ"
    }
  },
  "expected":{
    "curl":"curl -D- -u some_user:awesome_pass -X POST -H 'Content-Type: application/json' --data #body# https://jira.url/rest/api/2/issue/",
    "body":{
      "fields":{
        "project":{
          "key":"PROJ"
        },
        "issuetype":{
          "name":"Feature"
        },
        "summary":"_(no file content)_",
        "description":"_(no file content)_"
      }
    }
  }
}