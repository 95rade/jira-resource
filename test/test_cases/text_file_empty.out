{
  "params": {
    "text_file": "sample-empty.txt",
    "always_notify": "true",
    "debug": "true"
  },
  "source": {
      "url": "https://some.url",
      "username": "concourse",
      "password": "letmein"
    }
}
